src/sst/core/interprocess/mmapchild_pin3.h

```diff
@@ -0,0 +1,114 @@
+// Copyright 2009-2019 NTESS. Under the terms
+// of Contract DE-NA0003525 with NTESS, the U.S.
+// Government retains certain rights in this software.
+//
+// Copyright (c) 2009-2019, NTESS
+// All rights reserved.
+//
+// This file is part of the SST software package. For license
+// information, see the LICENSE file in the top level directory of the
+// distribution.
+
+#ifndef SST_CORE_INTERPROCESS_TUNNEL_MMAP_CHILD_PIN3_H
+#define SST_CORE_INTERPROCESS_TUNNEL_MMAP_CHILD_PIN3_H 1
+
+#include "pin.H"
+#include <iostream>
+#include <fstream>
+#include <sys/shm.h>
+#include <sys/stat.h>
+#include "atomic.hpp"
+
+#include "sst/core/interprocess/tunneldef.h"
```

Comment:
Why does this get included? The `TunnelDef` class doesn't seem to be used.

Reply:
Oops, I'll fix it.

Reply:
I remember now, it needs the size of InternalSharedData from the tunneldef class